src/test/java/com/dnd/modutime/core/room/integration/RoomTimeTableInitializerTest.java

```diff
@@ -70,4 +70,68 @@ public class RoomTimeTableInitializerTest {
         assertThat(counts).hasSize(2)
                 .allMatch(it -> it.equals(0));
     }
+
+    @Test
+    void 시작시간이_끝시간보다_작을때_해당_정보에_맞게_time_table을_세팅한다() {
+        // given
+        Room room = getRoom(_11_00, _13_00, List.of(_2023_02_09), 2);
+        roomRepository.save(room);
+        TimeTable timeTable = timeTableRepository.save(new TimeTable(room.getUuid()));
+        roomTimeTableInitializer.initialize(room.getUuid(), timeTable);
+
+        // when
+        List<TimeInfo> timeInfos = timeTable.getDateInfos().get(0).getTimeInfos();
+
+        // then
+        assertAll(
+            () -> assertThat(timeInfos.get(0).getTime().equals(_11_00)),
+            () -> assertThat(timeInfos.get(1).getTime().equals(_11_30)),
+            () -> assertThat(timeInfos.get(2).getTime().equals(_12_00)),
+            () -> assertThat(timeInfos.get(3).getTime().equals(_12_30))
```

Comment:
r: 리스트의 요소를 하나씩 꺼내기보다는 containsExactly로 검증해야할 것 같습니다
만약 4개가 아니라 5개가 들어있어도 해당 테스트는 성공할 것 같습니다.

```suggestion
        List<TimeInfo> timeInfos = timeTable.getDateInfos().get(0).getTimeInfos();
        List<LocalTime> times = timeInfos.stream()
                .map(TimeInfo::getTime)
                .collect(Collectors.toList());

        // then
        assertThat(times).hasSize(4)
                .containsExactly(_11_00, _11_30, _12_00, _12_30);
```

Reply:
와 하나 더 배워갑니다 역시 최고입니다